### PR TITLE
[Icon] add LocationIssue and LocationValid icons

### LINKED ISSUE #11938: [Icon]: New icon <LocationValid>
### How will these icons be used?

This icon will be used in all the places where we validate an address to indicate a 'valid' address. To name a few: 

- Orders
- Customers
- Companies
- Locations
- Shipping Labels
- Draft Orders

Frame 1321315522

Frame 1321315524

Frame 1321315525

Frame 1321315523

Frame 1321315526

### Icon name

LocationValid

### Icon description

Action: Used to denote the validity status of an address
Wayfinding: Used to help merchants understand that the address is valid
Concept: Used to represent the validity of an address
Education: Used to give merchants information about the validity of the address

### Icon keywords

location, address, shipping address, validation, issue, verification

### Icon author

Vihar Kothamasu

### Icon Figma link

[Figma for the icon](https://www.figma.com/file/m4W579SQGK5ylIk0VWARUC/Address-Validation-Iconography?type=design&node-id=2%3A5&mode=design&t=45fALx4n7F1vmaDL-1)

### Icon SVG code

```shell
<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M8.82655 16H5.75C5.33579 16 5 16.3358 5 16.75C5 17.1642 5.33579 17.5 5.75 17.5H14.25C14.6642 17.5 15 17.1642 15 16.75C15 16.3358 14.6642 16 14.25 16H11.1734C11.196 15.98 11.219 15.9596 11.2424 15.9387C11.707 15.5229 12.3273 14.9219 12.9495 14.1805C14.1737 12.7217 15.5 10.6071 15.5 8.2267C15.5 5.06038 13.1721 2.75 10 2.75C6.82792 2.75 4.5 5.06038 4.5 8.2267C4.5 10.6071 5.82627 12.7217 7.0505 14.1805C7.6727 14.9219 8.29304 15.5229 8.75765 15.9387C8.781 15.9596 8.80398 15.98 8.82655 16ZM10 4.25C7.64924 4.25 6 5.8959 6 8.2267C6 10.073 7.04873 11.845 8.1995 13.2162C8.7648 13.8898 9.33196 14.4397 9.75798 14.8209C9.84521 14.899 9.9263 14.9698 10 15.033C10.0737 14.9698 10.1548 14.899 10.242 14.8209C10.668 14.4397 11.2352 13.8898 11.8005 13.2162C12.9513 11.845 14 10.073 14 8.2267C14 5.8959 12.3508 4.25 10 4.25Z" fill="#4A4A4A"/>
  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.4801 6.92383C12.7983 7.18901 12.8413 7.66193 12.5762 7.98014L10.0762 10.9801C9.94127 11.142 9.74449 11.2397 9.53399 11.2492C9.32348 11.2588 9.11867 11.1793 8.96967 11.0303L7.46967 9.53033C7.17678 9.23744 7.17678 8.76256 7.46967 8.46967C7.76256 8.17678 8.23744 8.17678 8.53033 8.46967L9.4496 9.38894L11.4238 7.01986C11.689 6.70165 12.1619 6.65866 12.4801 6.92383Z" fill="#4A4A4A"/>
</svg>
```

### Are there any other important details?

I picked `LocationValid` as the icon name considering we already have `Location` and `LocationNone` as icons today which uses the same location pin.

### LINKED ISSUE #11939: [Icon]: New icon <LocationAlert>
### How will these icons be used?

This icon will be used in all the places where we validate an address to indicate a potentially 'invalid' address. The primary use case is found in the orders index

Default

Default-1

### Icon name

LocationAlert

### Icon description

Action: Used to denote the invalid status of an address
Wayfinding: Used to help merchants understand that the address is invalid and take action to fix it
Concept: Used to represent the invalidity of an address
Education: Used to give merchants information about the invalidity of the address

### Icon keywords

location, address, shipping address, validation, issue, verification, alert

### Icon author

Vihar Kothamasu

### Icon Figma link

[Figma for the icon](https://www.figma.com/file/m4W579SQGK5ylIk0VWARUC/Address-Validation-Iconography?type=design&node-id=1%3A58&mode=design&t=45fALx4n7F1vmaDL-1)

### Icon SVG code

```shell
<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none">
  <path fill-rule="evenodd" clip-rule="evenodd" d="M8.82655 16H5.75C5.33579 16 5 16.3358 5 16.75C5 17.1642 5.33579 17.5 5.75 17.5H14.25C14.6642 17.5 15 17.1642 15 16.75C15 16.3358 14.6642 16 14.25 16H11.1734C11.196 15.98 11.219 15.9596 11.2424 15.9387C11.707 15.5229 12.3273 14.9219 12.9495 14.1805C14.1737 12.7217 15.5 10.6071 15.5 8.2267C15.5 5.06038 13.1721 2.75 10 2.75C6.82792 2.75 4.5 5.06038 4.5 8.2267C4.5 10.6071 5.82627 12.7217 7.0505 14.1805C7.6727 14.9219 8.29304 15.5229 8.75765 15.9387C8.781 15.9596 8.80398 15.98 8.82655 16ZM10 4.25C7.64924 4.25 6 5.8959 6 8.2267C6 10.073 7.04873 11.845 8.1995 13.2162C8.7648 13.8898 9.33196 14.4397 9.75798 14.8209C9.84521 14.899 9.9263 14.9698 10 15.033C10.0737 14.9698 10.1548 14.899 10.242 14.8209C10.668 14.4397 11.2352 13.8898 11.8005 13.2162C12.9513 11.845 14 10.073 14 8.2267C14 5.8959 12.3508 4.25 10 4.25Z" fill="#4A4A4A"/>
  <path d="M9.25 6.75C9.25 6.33579 9.58579 6 10 6C10.4142 6 10.75 6.33579 10.75 6.75V9.25C10.75 9.66421 10.4142 10 10 10C9.58579 10 9.25 9.66421 9.25 9.25V6.75Z" fill="#4A4A4A"/>
  <path d="M10.75 11.75C10.75 12.1642 10.4142 12.5 10 12.5C9.58579 12.5 9.25 12.1642 9.25 11.75C9.25 11.3358 9.58579 11 10 11C10.4142 11 10.75 11.3358 10.75 11.75Z" fill="#4A4A4A"/>
</svg>
```

### Are there any other important details?

I picked `LocationAlert` as the icon name considering we already have `Location` and `LocationNone` as icons today which uses the same location pin.

### PR DESCRIPTION
**Pull request by JordanForeman**
### WHY are these changes introduced?

resolves https://github.com/Shopify/polaris/issues/11939
resolves https://github.com/Shopify/polaris/issues/11938

### WHAT is this pull request doing?

Adds a new icons to the `polaris-icons` package. 

cc. @ksvihar 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#install-dependencies-and-build-workspaces)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/.github/CONTRIBUTING.md#changelog)

### 🎩 checklist

- [ ] Tested a [snapshot](https://github.com/Shopify/polaris/blob/main/documentation/Releasing.md#-snapshot-releases)
- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide

### CODE REVIEW

**Review by sethomas — APPROVED** [review]
LGTM, might want to wait for a polaris engineer to approve!

File: polaris-icons/icons/LocationValidIcon.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: LocationValid
+description: |
+  Action: Used to denote the validity status of an address
+  Wayfinding: Used to help merchants understand that the address is valid
+  Concept: Used to represent the validity of an address
+  Education: Used to give merchants information about the validity of the address
```
Comment (sam-b-rose) [inline polaris-icons/icons/LocationValidIcon.yml]:
```suggestion
description: Used to represent the validity of an address.
```

File: polaris-icons/icons/LocationAlertIcon.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: LocationAlert
+description: |
+  Action: Used to denote the invalid status of an address
+  Wayfinding: Used to help merchants understand that the address is invalid and take action to fix it
+  Concept: Used to represent the invalidity of an address
+  Education: Used to give merchants information about the invalidity of the address
```
Comment (sam-b-rose) [inline polaris-icons/icons/LocationAlertIcon.yml]:
```suggestion
description: Used to represent the invalidity of an address.
```

File: polaris-icons/icons/LocationAlertIcon.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: LocationAlert
```
Comment (sam-b-rose) [inline polaris-icons/icons/LocationAlertIcon.yml]:
```suggestion
name: Location alert
```

File: polaris-icons/icons/LocationValidIcon.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+name: LocationValid
```
Comment (sam-b-rose) [inline polaris-icons/icons/LocationValidIcon.yml]:
```suggestion
name: Location valid
```

**Review by sam-b-rose — COMMENTED** [review]
Looks good! I've added a commit that cleans up the icon paths using the `optimize` script.

Also a few consistency suggestions to closer match the conventions in other icon YAML files.

**Comment by JordanForeman** [comment]
@sam-b-rose thanks for the suggestions and optimizations! I've incorporated your feedback on the `yml` files :smile:

**Comment by alex-page** [comment]
@JordanForeman what is the timeline/urgency for these icons? We are building a new icon library to replace this one.

**Review by alex-page — CHANGES_REQUESTED** [review]
@JordanForeman what is the timeline/urgency for these icons? We are building a new icon library to replace this one.

**Comment by ksvihar** [comment]
I just realised that the location icons are updated since we first created the icons. So, I just updated the icons and respective issues with to reflect the latest icon designs.

**Comment by JordanForeman** [comment]
@alex-page I'd say "medium" urgency? We're aiming for mid-May to start using this. I'm not sure we'll have time to include migration to a new icon library, depending on the costs associated with that.

**Comment by JordanForeman** [comment]
@ksvihar updated! See [0260716](https://github.com/Shopify/polaris/pull/11940/commits/0260716e429394af071936c91efb67af7d5d36e9)

**Comment by alex-page** [comment]
@JordanForeman is there any blockers or things stopping you from using the icons without polaris-icons until the new library is implemented?

**Comment by JordanForeman** [comment]
@alex-page nope :smile: I can just hardcode these for now, and we can keep tabs on how icons evolves.

**Pull request closed without merging**